src/main/java/com/outbrain/ci/friendly/flatten/maven/plugin/FlattenMojo.java

```diff
@@ -76,18 +77,18 @@ private File writePom(final String content) throws MojoExecutionException {
 
     final File parentFile = flattenedPomFile.getParentFile();
     if (!parentFile.exists()) {
-      boolean success = parentFile.mkdirs();
+      final boolean success = parentFile.mkdirs();
       if (!success) {
         throw new MojoExecutionException("Failed to create directory " + flattenedPomFile.getParent());
       }
     }
 
-    try (FileWriter writer = new FileWriter(flattenedPomFile)) {
+    try (final FileWriter writer = new FileWriter(flattenedPomFile)) {
       writer.write(content);
-      getLog().info("Successfully wrote to the file.");
-    } catch (IOException e) {
+      getLog().info("Successfully wrote to " + flattenedPomFile.getAbsolutePath());
```

Comment:
let's remove the getAbsolutePath to be consistent with the error log below

Reply:
Aligned